test/files/pos/t6734.scala

```diff
@@ -0,0 +1,23 @@
+
+//single file badimp.scala
+// adding package object gives not found: type SortedMap
+package object badimp
```

Comment:
The funny thing is that it's not the package object, but the package introduced by desugaring that received the synthetic.